staging/kos/pkg/controllers/connectionagent/connectionagent.go

```diff
@@ -432,13 +432,14 @@ func (ca *ConnectionAgent) onLocalAttAdded(obj interface{}) {
 	ca.queue.Add(parse.AttNSN(att))
 }
 
-func (ca *ConnectionAgent) onLocalAttUpdated(oldObj, newObj interface{}) {
-	oldAtt := oldObj.(*netv1a1.NetworkAttachment)
-	newAtt := newObj.(*netv1a1.NetworkAttachment)
-	glog.V(5).Infof("Local NetworkAttachments cache: notified of update from %#+v to %#+v",
-		oldAtt,
-		newAtt)
-	ca.queue.Add(parse.AttNSN(newAtt))
+func (ca *ConnectionAgent) onLocalAttUpdated(oldObj, obj interface{}) {
+	oldAtt, att := oldObj.(*netv1a1.NetworkAttachment), obj.(*netv1a1.NetworkAttachment)
+	glog.V(5).Infof("Local NetworkAttachments cache: notified of update from %#+v to %#+v", oldAtt, att)
+	if oldAtt.Status.IPv4 != att.Status.IPv4 || oldAtt.Status.AddressVNI != att.Status.AddressVNI {
```

Comment:
For efficiency enqueue only if the update can lead to a change of the interface. This was possible even without immutability, but the if condition would have been longer.